tfe/data_source_organization.go

```diff
@@ -0,0 +1,168 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEOrganization() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEOrganizationRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"external_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"collaborator_auth_policy": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"cost_estimation_enabled": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"enterprise_plan": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"owners_team_saml_role_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"permissions": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"can_create_team": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_create_workspace": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_create_workspace_migration": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_destroy": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_traverse": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_update": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_update_api_token": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_update_oauth": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+
+						"can_update_sentinel": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"saml_enalbed": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
```

Comment:
This attribute is essentially deprecated. See the [internal!] serialization logic for more on what this is and why.